webapp/src/Utils/Scoreboard/Filter.php

```diff
@@ -4,28 +4,19 @@
 
 class Filter
 {
-    /** int[] */
-    public array $affiliations = [];
-
-    /** @var string[] */
-    public array $countries = [];
-
-    /** @var int[] */
-    public array $categories = [];
-
-    /** @var int[] */
-    public array $teams = [];
-
+    /**
+     * @param int[] $affiliations
+     * @param string[] $countries
+     * @param int[] $categories
+     * @param int[] $teams
+     */
     public function __construct(
-        array $affiliations = [],
-        array $countries = [],
-        array $categories = [],
-        array $teams = []
-    ) {
-        $this->affiliations = $affiliations;
-        $this->countries    = $countries;
-        $this->categories   = $categories;
-        $this->teams        = $teams;
+        public array $affiliations = [],
+        public array $countries = [],
+        public array $categories = [],
+        public array $teams = []
+    )
+    {
     }
```

Comment:
```suggestion
    ) {}
```

Reply:
We should make a decision on
```
    ) {
    }
```
(which is officially what PSR2/PSR12 want)

vs
```
    ) {}
```
(which saves a line and doesn't confuse too much)

Im fine with either (and PHPStorm has an explicit setting for this).

Reply:
I think we should for those situations go with {}, it makes it explicit that nothing happens whereas `{\n}` could be a forgotten statement.

Reply:
I do dislike it if we can't make phpcs allow it (and only it). Could you investigate? I'm already bummed that php-cs-fixer just won't do it, but if phpcs can, that's at least something.

Reply:
I completely agree that we should use `{}`. This is really clearer, and I think a lack of tooling support would be a bad reason to not use it.